Dockerfile

```diff
@@ -0,0 +1,39 @@
+# Use the official Golang image to build the Go binary
+FROM golang:1.22 AS builder
+
+# Set the Current Working Directory inside the container
+WORKDIR /app
+
+# Copy go.mod and go.sum files and download dependencies
+COPY go.mod go.sum ./
+RUN go mod download
+
+# Copy the source code into the container
+COPY . .
+
+# Build the Go app
+RUN GOOS=linux GOARCH=amd64 go build -ldflags="-s" -o main cmd/productivite/main.go
```

Comment:
<div>


<div id="suggestion">
<div id="issue"><b>Consider adding version info in binary</b></div>
<div id="fix">

Consider adding version information to the binary using `-ldflags`. Could add `-X main.Version=${VERSION}` to embed version info at build time.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<blockquote>Check the AI-generated fix before applying</blockquote>
<div id="code">


```suggestion
ARG VERSION=dev
RUN GOOS=linux GOARCH=amd64 go build -ldflags="-s -X main.Version=${VERSION}" -o main cmd/productivite/main.go
```

</div>
</details>
</div>


<small><i>Code Review Run <a href=https://github.com/aakashgusain1990/productivite/pull/14#issuecomment-2563464674>#148ccb</a></i></small>
</div>

---
Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged